easybuild/easyconfigs/s/SimNIBS/SimNIBS-4.0.1-foss-2023a.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'PythonBundle'
+
+name = 'SimNIBS'
+version = '4.0.1'
+
+homepage = 'https://simnibs.github.io/simnibs'
+description = "SimNIBS is a free and open source software package for the Simulation of Non-invasive Brain Stimulation"
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [('CMake', '3.26.3')]
+dependencies = [
+    ('Python', '3.11.3'),
+    ('Python-bundle-PyPI', '2023.06'),
+    ('SciPy-bundle', '2023.07'),
+    ('matplotlib', '3.7.2'),
+    ('Pillow', '10.0.0'),
+    ('h5py', '3.9.0'),
+    ('PyOpenGL', '3.1.7'),
+    ('freeglut', '3.4.0'),
+    ('Boost', '1.74.0'),
+    ('tbb', '2020.1'),
+    ('libwebp', '1.3.1'),
+    ('NiBabel', '5.2.0'),
+    ('PyQt5', '5.15.10'),
+    ('FMM3D', '1.0.4'),
+    ('charm-gems', '1.3.3'),
+    ('CGAL', '5.4'),
+    ('Eigen', '3.3.7', '', SYSTEM),
```

Comment:
This could be probably be:

```
('Eigen', '3.4.0'),
```

and Eigen should always be a build dependency, because it's a header-only library